assets/postcss.config.js

```diff
@@ -1,15 +1,12 @@
-const autoprefixerOptions = [
-    '> 2%',
-    'Last 2 versions',
-    'IE 11',
-    'not dead'
-];
+const autoprefixerOptions = {
+    remove: false
+};
 
 module.exports = ctx => ({
     map: Object.assign({}, ctx.options.map, {inline: false}),
     parser: ctx.options.parser,
     plugins: {
-        autoprefixer: autoprefixerOptions,
+        'autoprefixer': autoprefixerOptions,
```

Comment:
Was this the change that fixed things? Or was it the 'remove: false'?

Reply:
remove:false fixed it. I just tried to make things more consistent in the config file.